monGARS/core/embeddings.py

```
@@ -89,6 +101,202 @@ class EmbeddingBackendError(RuntimeError):
 )
 
 
+def _resolve_transformers_device(
+    settings: Settings,
+):  # noqa: ANN201 - torch device runtime
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `ANN201`)

Remove unused `noqa` directive

(RUF100)